src/microbe_stage/editor/MicrobeEditor.tscn

```diff
@@ -3927,6 +3927,25 @@ __meta__ = {
 "_edit_use_anchors_": false
 }
 
+[node name="ErrorMessage" type="MarginContainer" parent="MicrobeEditorGUI/CellEditor"]
```

Comment:
I think this node should be named a bit more sensibly, as each different purpose should have it's own popup.
Also this should use the popup with ok Control type instead of being a MarginContainer.

Reply:
Ingame you can see, I tries to make a universal and reuseable error message that can occur above the finish button
![image](https://user-images.githubusercontent.com/30809803/95742643-9e901a00-0c90-11eb-8508-a6b325b36c77.png)

Reply:
As I mentioned before I think it should be a popup error only being shown if you attempt to finish. So that it has enough space to include info about undoing stuff if you are out of MP, and in the future once we have organelle moving it can contain that info as well.